### PR TITLE
Init clean up

### PR DESCRIPTION
**Pull request by plbonneville**

### CODE REVIEW

File: CoreMVC/appsettings.Development.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Debug",
-      "System": "Information",
-      "Microsoft": "Information"
+      "Default": "Information",
+      "Microsoft": "Warning",
```
Comment (joshcampbell191) [inline CoreMVC/appsettings.Development.json:5]: Revert these settings

File: CoreMVC/appsettings.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Warning"
+      "Default": "Information",
```
Comment (joshcampbell191) [inline CoreMVC/appsettings.json:4]: Revert this setting

File: CoreMVC/Startup.cs [hunk 3]
```diff
@@ -45,34 +49,33 @@ public void ConfigureServices(IServiceCollection services)
...
         {
-            app.UseRequestLocalization();
-
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
             else
             {
+                //app.UseExceptionHandler("/Home/Error");
```
Comment (joshcampbell191) [inline CoreMVC/Startup.cs:63]: Please remove as discussed.

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.Web;
+using System.Web.Mvc;
+
+namespace GoC.WebTemplate.MVC
+{
+    public class TestLayoutsCdtsTemplateActionFilterAttribute : CdtsTemplateActionFilterAttribute
```
Comment (joshcampbell191) [inline MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:8]: Was this added to provide backwards compatibility for to support the "?env=...".

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        public override void OnActionExecuting(ActionExecutingContext filterContext)
+        {
+            SetEnvironment(filterContext);
+
+            base.OnActionExecuting(filterContext);
+        }
+
+        private void SetEnvironment(ActionExecutingContext filterContext)
+        {
+            var components = HttpUtility.ParseQueryString(filterContext.RequestContext.HttpContext.Request.QueryString.ToString().ToUpperInvariant());
```
Comment (joshcampbell191) [inline MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:19]: Is it possible to use `filterContext.ActionParameters` for this?

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            foreach (var item in components.Keys)
+            {
+                if (item.ToString() == "ENV")
+                {
+                    return components.Get(item.ToString());
+                }
+            }
+
+            return null;
```
Comment (joshcampbell191) [inline MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:39]: Can we replace this with a LINQ expression?

File: CoreMVC/Startup.cs (hunk 3 above)
Comment (plbonneville) [reply CoreMVC/Startup.cs:63]: Removed.

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs (hunk 4 above)
Comment (plbonneville) [reply MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:8]: Yes, this ActionFilterAttribute was added only to mimic the TestLayoutsController. Is isn't needed and can be removed.

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs (hunk 5 above)
Comment (plbonneville) [reply MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:19]: The class was removed.

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs (hunk 4 above)
Comment (plbonneville) [reply MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:8]: The class was removed.

File: MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs (hunk 6 above)
Comment (plbonneville) [reply MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:39]: No, as the class didn't implement the generic version of IEnumerable.

Comment (plbonneville) [reply MVC/CdtsTemplateActionFilterAttribute.TestLayouts.cs:39]: The class was removed.

File: CoreMVC/appsettings.Development.json (hunk 1 above)
Comment (plbonneville) [reply CoreMVC/appsettings.Development.json:5]: I don't think we should revert the settings.

The modification was made to revert to the default settings offered when creating a new ASP.NET Core 3.1 project.

File: CoreMVC/appsettings.json (hunk 2 above)
Comment (plbonneville) [reply CoreMVC/appsettings.json:4]: I don't think we should revert the settings.

The modification was made to revert to the default settings offered when creating a new ASP.NET Core 3.1 project.

**Pull request merged**